lib/xnvme_be_ramdisk_sync.c

```diff
@@ -46,6 +46,10 @@ xnvme_be_ramdisk_sync_cmd_io(struct xnvme_cmd_ctx *ctx, void *dbuf, size_t dbuf_
 	case XNVME_SPEC_FS_OPC_FLUSH:
 		break;
 
+	case XNVME_SPEC_NVM_OPC_DATASET_MANAGEMENT:
```

Comment:
I think this can be moved up to the to `XNVME_SPEC_NVM_OPC_FLUSH` case, the comment is implied and I don't think there's any reason to have a separate case just for that

Reply:
I think it's best to keep them separate, as it could also imply that DSM is related or should be handled the same as a flush. Maybe the comment is trivial if you know about DSM, but it can't hurt to tell the reader why we are skipping on it.

Reply:
That's a good point.